certbot-dns-route53/certbot_dns_route53/dns_route53.py

```diff
@@ -42,14 +42,26 @@ def more_info(self):  # pylint: disable=missing-docstring,no-self-use
     def _setup_credentials(self):
         pass
 
-    def _perform(self, domain, validation_domain_name, validation):
-        try:
-            change_id = self._change_txt_record("UPSERT", validation_domain_name, validation)
+    def _perform(self, achalls): # pylint: disable=missing-docstring
```

Comment:
The parameters here should be `self`, `domain`, `validation_domain_name`, and `validation` to make the linter happy.